src/ui-kit/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { action } from '@storybook/addon-actions'
+import { Formik } from 'formik'
+import React from 'react'
+
+import TextInput from './TextInput'
+
+export default {
+  title: 'ui-kit/TextInput',
+  component: TextInput,
+}
+
+const Template = () => (
+  <Formik
+    initialValues={{ email: '' }}
+    onSubmit={action('onSubmit')}
+  >
+    {({ values, handleChange, handleBlur }) => {
+      return (
+        <TextInput
+          name='email'
+          onBlur={handleBlur}
+          onChange={handleChange}
+          value={values.email}
+        />
+      )}}
```

Comment:
d'une manière générale (valable pour la plutart des champs), je pense qu'on va plutôt utiliser 
```suggestion
    {({ getFieldProps }) => {
      return (
        <TextInput {...getFieldProps('email')}  />
      )}}
```